docker-build.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh -ex
+# Build a calicokube image containing cni and ipam
+# for a given K8S VERSION env or $1 arg
+
+K8S="${1:-1.3.1}"
+K8S=${K8S:-$K8S_VERSION}
+docker build --build-arg K8S_VERSION=$K8S -t calicokube .
```

Comment:
I think this logic could be moved into a `make` target, so you can simply "make docker-image" or something similar to trigger the build.